src/main/java/seedu/address/logic/commands/DeleteTagCommand.java

```diff
@@ -0,0 +1,42 @@
+package seedu.address.logic.commands;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.person.exceptions.DuplicatePersonException;
+import seedu.address.model.person.exceptions.PersonNotFoundException;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Deletes a tag from all contacts in the address book.
+ */
+public class DeleteTagCommand extends UndoableCommand {
+
+    public static final String COMMAND_WORD = "tagDelete";
+    public static final String COMMAND_ALIAS = "t";
```

Comment:
I don't think "t" is a clear label for tagDelete, perhaps using "td" would be better?